jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -617,3 +617,34 @@ teardown() {
     echo "GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE}"
     [[ "${GC_MAX_METASPACE_SIZE}" == "${KIE_SERVER_MAX_METASPACE_SIZE}" ]]
 }
+
+@test "Verify the Kafka extension" {
+    export KIE_SERVER_KAFKA_EXT_ENABLED="true"
+    export KIE_SERVER_KAFKA_EXT_BOOTSTRAP_SERVERS="localhost:9092"
+    export KIE_SERVER_KAFKA_EXT_CLIENT_ID="app"
+    export KIE_SERVER_KAFKA_EXT_GROUP_ID="jbpm-consumer"
+    export KIE_SERVER_KAFKA_EXT_ACKS="2"
+    export KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS="2000"
+    export KIE_SERVER_KAFKA_EXT_AUTOCREATE_TOPICS="true"
+    export KIE_SERVER_KAFKA_EXT_TOPICS="{person=human} {dog=animal}"
```

Comment:
why use brackets?
it is simpler just to person=human, dog=animal, and on the script comma as field separator. imho this way just adds an unnecessary layer of complexity, plus, the description on the template does not match this behavior.

Reply:
Aren't used on the operator when packed inside an array ?

Reply:
> Aren't used on the operator when packed inside an array ?

?

Reply:
Fixed